examples/plotting_with_folium.ipynb

```diff
@@ -0,0 +1,457 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plotting with folium\n",
+    "\n",
+    "__What is Folium?__\n",
+    "\n",
+    "It builds on the data wrangling and a Python wrapper for leaflet.js. It makes it easy to visualize data in Python with minimal instructions.\n",
+    "\n",
+    "Let's see the implementation of both GeoPandas and Folium:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Importing Libraries\n",
+    "import pandas as pd\n",
+    "import geopandas as gpd\n",
```

Comment:
Can you use the import `import geopandas` instead of `import geopandas as gpd` (so without alias). To ensure consistency within our documentation.